### PR TITLE
Fix warning options on windows

### PR DESCRIPTION
**Pull request by deadeyegoodwin**

### CODE REVIEW

**Review by GuanLuo — APPROVED** [review]

File: qa/L0_batcher/batcher_test.py [hunk 1]
```diff
@@ -38,6 +38,10 @@
 
 import tritonclient.grpc as grpcclient
 
+# By default, find tritonserver on "localhost", but can be overridden
+# with TRITONSERVER_IPADDR envvar
+_tritonserver_ipaddr = os.environ.get('TRITONSERVER_IPADDR', 'localhost')
```
Comment (CoderHam) [inline qa/L0_batcher/batcher_test.py:43]: We would need to make this change for the gitlab scripts that are used for windows testing as well. For example L0_onnx_execution_provider and L0_openvino_models?

File: qa/L0_sequence_batcher/test.sh [hunk 2]
```diff
@@ -82,7 +82,7 @@ TF_VERSION=${TF_VERSION:=1}
 # On windows the paths invoked by the script (running in WSL) must use
 # /mnt/c when needed but the paths on the tritonserver command-line
 # must be C:/ style.
-if [[ "$(< /proc/sys/kernel/osrelease)" == *Microsoft ]]; then
```
Comment (CoderHam) [inline qa/L0_sequence_batcher/test.sh:85]: This as well needs to be changed for gitlab side

File: qa/common/infer_util.py [hunk 3]
```diff
@@ -24,6 +24,7 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
```
Comment (CoderHam) [inline qa/common/infer_util.py:24]: Update copyright

File: qa/common/libtorch_infer_client.py [hunk 4]
```diff
@@ -25,6 +25,7 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
```
Comment (CoderHam) [inline qa/common/libtorch_infer_client.py:25]: Here as well

**Review by CoderHam — CHANGES_REQUESTED** [review]
Update copyrights where neeed

File: qa/L0_batcher/batcher_test.py (hunk 1 above)
Comment (deadeyegoodwin) [reply qa/L0_batcher/batcher_test.py:43]: The gitlab MR should have all the necessary changes.

File: qa/L0_sequence_batcher/test.sh (hunk 2 above)
Comment (deadeyegoodwin) [reply qa/L0_sequence_batcher/test.sh:85]: See the updated MR

**Review by CoderHam — APPROVED** [review]

**Pull request merged**